english/God's-own-country/44rivers/kerala1.md

```diff
@@ -0,0 +1,2 @@
+Kerala, a state on India's tropical Malabar Coast, has nearly 600km of Arabian Sea shoreline.
+It's known for its palm-lined [beaches](beaches/kerala2.md) 
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.